contrib/resources/translations/README.md

```diff
@@ -13,25 +13,27 @@ dosbox -lang en -c "config -wl en.lng" -c "exit"
 or do the following:
 
 1. In the configuration file, go to the `[dosbox]` section and set`language = en`.
-2. Run the most current version of DOSBox Staging.
-3. Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
+1. Run the most current version of DOSBox Staging.
+1. Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
```

Comment:
You have to use incrementing number or the linter will object.

Reply:
So we have two markdown linters giving contradicting remarks... OK, changed.